src/Operation/Reject.php

```diff
@@ -26,7 +26,7 @@ final class Reject extends AbstractOperation
     /**
      * @pure
      *
-     * @return Closure(callable(T, TKey, Iterator<TKey, T>):bool...): Closure(Iterator<TKey, T>): Generator<int, list<array{0: TKey, 1: T}>>
+     * @return Closure(callable(T , TKey, Iterator<TKey, T>): bool ...): Closure (Iterator<TKey, T>): Generator<TKey, T>
```

Comment:
Ok I won't complain about extra spaces, but we should fix them.

Reply:
no you're right there were a couple that were wrong, fixed now

Reply:
Thanks :)